keyboards/planck/keymaps/am/keymap.c

```diff
@@ -0,0 +1,261 @@
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+// this is the style you want to emulate.
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _LOWER 2
+#define _RAISE 3
+#define _TOUCHCURSOR 4
+#define _MOUSE 5
+#define _ADJUST 16
+
+enum planck_keycodes {
+  QWERTY = SAFE_RANGE,
+  COLEMAK,
+  LOWER,
+  RAISE,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+// Taps
+#define CTL_ESC CTL_T(KC_ESC)            // Tap for Esc, hold for Ctr
+#define SFT_ENT SFT_T(KC_ENT)            // Tap for Enter, hold for Shift
+#define LT_TC   LT(_TOUCHCURSOR, KC_SPC) // L-ayer T-ap T-ouch C-ursor
+#define LT_ML   LT(_MOUSE, KC_A)         // L-ayer T-ap M-ouse C-ursor (on A)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+ * ,--------------------------------------------------------------------------------------.
+ * | Tab    |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp  |
+ * |--------+------+------+------+------+-------------+------+------+------+------+-------|
+ * |Ctrl/Esc| ML/A |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |   "   |
+ * |--------+------+------+------+------+------|------+------+------+------+------+-------|
+ * | Shift  |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Sft/Ent|
+ * |--------+------+------+------+------+------+------+------+------+------+------+-------|
+ * | Power  |   ~  | Alt  | GUI  |Lower |   TC/Space  |Raise | Next | Vol- | Vol+ |  Play |
+ * `--------------------------------------------------------------------------------------'
+ */
+[_QWERTY] = {
```

Comment:
You may want to update the layer blocks to use the newer format: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L58-L63